shared/utils/dateFilterUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { DateTime } from 'luxon'
+
+import { EventModel } from '../api'
+import { MAX_DATE_RECURRENCES } from '../constants'
+
+const isWithinDateRange = (
+  startDate: DateTime | null,
+  endDate: DateTime | null,
+  itemStartDate: DateTime,
```

Comment:
🙃 I think these should be called `eventStartDate` and `eventEndDate`, they're not items